services/etcd.go

```diff
@@ -256,6 +257,34 @@ func RunEtcdSnapshotSave(ctx context.Context, etcdHost *hosts.Host, prsMap map[s
 		imageCfg.Cmd = append(imageCfg.Cmd, "--retention="+retention)
 		imageCfg.Cmd = append(imageCfg.Cmd, "--creation="+creation)
 	}
+	// boolean flag to export to AWS S3 is true -> set parameters for login and backup location
+	if s3 {
+		imageCfg.Cmd = append(imageCfg.Cmd, "--s3")
+		imageCfg.Cmd = append(imageCfg.Cmd, "--s3bucket="+s3bucket)
```

Comment:
I think you should add some kind of validation in validation code to make sure that the user passes at least the s3bucket if s3 option is passed